tests/fixtures/wallet_transaction.json

```diff
@@ -15,6 +16,7 @@
       "lago_id": "b7ab2926-1de8-4428-9bcd-779314ac1222",
       "lago_wallet_id": "12345",
       "status": "settled",
+      "source": "manual",
```

Comment:
Maybe you could add some metadata in this file to validates it is correctly mapped in the python object